app/models/scimitar/resources/mixin.rb

```diff
@@ -335,25 +335,16 @@ def scim_queryable_attributes
           @scim_queryable_attributes ||= self.class.scim_queryable_attributes()
         end
 
-        # Render self as a SCIM object using ::scim_attributes_map.
+        # Render self as a SCIM object using ::scim_attributes_map. Fields that
+        # are marked as <tt>returned: 'never'</tt> are excluded.
         #
         # +location+:: The location (HTTP(S) full URI) of this resource, in the
         #              domain of the object including this mixin - "your" IDs,
         #              not the remote SCIM client's external IDs. #url_for is a
         #              good way to generate this.
         #
         def to_scim(location:)
-          map             = self.class.scim_attributes_map()
-          timestamps_map  = self.class.scim_timestamps_map() if self.class.respond_to?(:scim_timestamps_map)
-          attrs_hash      = self.to_scim_backend(data_source: self, attrs_map_or_leaf_value: map)
-          resource        = self.class.scim_resource_type().new(attrs_hash)
-          meta_attrs_hash = { location: location }
-
-          meta_attrs_hash[:created     ] = self.send(timestamps_map[:created     ])&.iso8601(0) if timestamps_map&.key?(:created)
-          meta_attrs_hash[:lastModified] = self.send(timestamps_map[:lastModified])&.iso8601(0) if timestamps_map&.key?(:lastModified)
-
-          resource.meta = Meta.new(meta_attrs_hash)
-          return resource
+          return self.to_scim_for_returning(location: location)
```

Comment:
This gives away that we can't actually fully avoid internally the concept of to-SCIM in "render normally" vs "render everything" mode; more info in later diff comments.